Add info about --global config options for Astro CLI (#3095)
astronomer/docs: astro/cli/configure-cli.md

## Set a configuration

You can set Astro configureations at two different scopes: globally or by project.

> For project-specific configurations, Astro stores the settings in a file named `/...//.astro/config.yaml` in your project directory. This file is generated when you run `astro dev init` in your project folder. Global configurations, however, are stored in a central location. In Linux, for examples, the configurations are stored in `/home//.astro/config.yaml`.

Author: @odaneau-astro - do we know the location for Mac/windows? Should I ask the CLI team?

Reviewer: You can't use the Astro CLI on Windows directly. You have to use WSL / Linux.
On Mac, it should be in ~/.astro/config.yaml. 
I can't confirm since I'm on Windows, but it's easy to check with `ls -la`.

## Set a configuration

You can set Astro configureations at two different scopes: globally or by project.

> For project-specific configurations, Astro stores the settings in a file named `/...//.astro/config.yaml` in your project directory. This file is generated when you run `astro dev init` in your project folder. Global configurations, however, are stored in a central location. 

Reviewer 2: ```suggestion
For project-specific configurations, Astro stores the settings in a file named `.astro/config.yaml` in your project directory. This file is generated when you run `astro dev init` in your project folder. Global configurations, however, are stored in a central location with the Astro executable.  
```

Reviewer: Just to clarify this one: when installed manually from GH releases the Astro executable can be located anywhere on PATH. on the other hand, the astro global config location will not change.

> For project-specific configurations, Astro stores the settings in a file named `.astro/config.yaml` in your project directory. This file is generated when you run `astro dev init` in your project folder. Global configurations, however, are stored in a central location with the Astro executable.  

Reviewer 2: ```suggestion
For project-specific configurations, Astro stores the settings in a file named `.astro/config.yaml` in your project directory. This file is generated when you run `astro dev init` in your project folder. Global configurations, however, are stored in a central location with the Astro executable. If a configuration is set in a project configuration file, it always overrides configurations the same configuration in the global configuration file.
```

astro config set webserver.port 8081
```

### Set global configurations

> You can view or modify the `astro/config.yaml` file for your current astro projects by using the `--global` flag with `astro config set [option] --global` and `astro config get`. Setting a configuration globally applies it across projects, and the configuration is saved in a central location. For example, the configurations are stored in `~/.astro/config.yaml` on Mac OS and `/home//.astro/config.yaml` on Linux.

Reviewer 2: ```suggestion
You can view or modify the default configuration for all Astro projects using the `--global` flag. Setting a configuration globally applies it across all current and new projects, and the configuration is saved in a central location. For example, the configurations are stored in `~/.astro/config.yaml` on Mac OS and `/home//.astro/config.yaml` on Linux. You can then override the global configuration at the project level by running `astro config set` without the flag.
```

Reviewer 2: Something that might be worth clarifying: if you already have a config.yaml file set on a given project, and you update a config globally, is that config also applied to the project or does the local file still take precedence?

Reviewer: @sunkickr : Could you help answer @jwitz here?

> You can view or modify the `astro/config.yaml` file for your current astro projects by using the `--global` flag with `astro config set [option] --global` and `astro config get`. Setting a configuration globally applies it across projects, and the configuration is saved in a central location. For example, the configurations are stored in `~/.astro/config.yaml` on Mac OS and `/home//.astro/config.yaml` on Linux.

Reviewer 2: ```suggestion
Use the `--global` flag with `astro config set [option] --global` to set a configuration for all current and new Astro projects on your machine. Your global Astro CLI configuration is saved in a central location on your machine. For example, the configurations are stored in `~/.astro/config.yaml` on Mac OS and `/home//.astro/config.yaml` on Linux.

Note that you can override global configurations by setting a [project-level configuration](#set-project-configuration). If an Astro project contains an `.astro/config.yaml` file, any configuration values in that file take precedence over the same global configuration values.
```

> The following example shows how to set a port for your local Airflow webserver to `8081` from the default of `8080` for all your Astro projects:

Reviewer 2: ```suggestion
The following example shows how to set the local Airflow webserver port to `8081` for all Astro projects on your local machine:
```
